assets/scripts/segments/view.js

```diff
@@ -261,11 +304,13 @@ export function drawSegmentContents (ctx, type, variantString, actualWidth, offs
       const sprite = getSpriteDef(sprites[l])
       const svg = images.get(sprite.id)
       const center = dimensions.center
+
       const x = (center - ((svg.width / TILE_SIZE_ACTUAL) / 2) - left - (sprite.offsetX / TILE_SIZE_ACTUAL || 0)) * TILE_SIZE * multiplier
+      const distanceFromGround = multiplier * TILE_SIZE * ((svg.height - (sprite.originY || 0)) / TILE_SIZE_ACTUAL)
 
       drawSegmentImage(sprite.id, ctx, undefined, undefined, undefined, undefined,
         offsetLeft + x,
-        offsetTop + (multiplier * TILE_SIZE * (sprite.offsetY || 0)),
+        groundLevel - distanceFromGround,
         undefined, undefined, multiplier, dpi)
     }
   }
```

Comment:
I can't add a note to line 319 (`drawProgrammaticPeople()` but this function is hardcoding the Y-value at which we draw people. We should also pass the ground level number into that function so that it draws people at the right Y-level. If it's too hard to do in this PR, let's add a TODO note and make a follow up issue for it instead.

Reply:
This shouldn't be too hard to do. However, I did notice that in `people.json` we are still hardcoding the `width` value instead of using the `svg.width` value. Should I resolve this in a separate PR so I can remove the width as well?

Reply:
Oh never mind, the `width` value in `people.json` is not the same as the `svg.width` value?

Reply:
Hmm, I forgot about that `width` value. That one really is a little more awkward because I think all the people SVGs have the same intrinsic width, and this value is trying to say how much room each person should take up. It may make more sense to resize the SVGs so that we still don't need to hard-code width values... let's put this in another PR.

Reply:
@louh so I redefined `GROUND_LEVEL_OFFSETY` values so that `ASPHALT` has a lower value than `CURB`. However, by doing so I am no longer `offsetTop` and am instead using a new value that I am calling the `groundBaseline` which is essentially where elevation level 0 sits on the canvas. I haven't removed `offsetTop` from the parameters of `drawSegmentContents` because `drawProgrammaticPeople` is hardcoding the Y-value. 

I can remove the hardcoded Y-value in this PR and therefore remove `offsetTop`, or would you rather I do both in a separate PR along with removing the `width` value?

Reply:
If removing the hard-coded Y-value and `offsetTop` from `drawProgrammaticPeople` is easy to do here, let's do it in this PR, and remove `width` later. (Removing width from people may require changing the image assets, which is why it'll need to be a separate PR regardless.)